fathom/fnode.mjs

```diff
@@ -41,7 +41,7 @@ export class Fnode {
     }
 
     /**
-     * Return whether the given type is one of the ones attached to the node.
+     * Return whether the given type is one of the ones attached to the fnode.
```

Comment:
I really was talking about the HTML node from here down. Maybe we say "HTML node I wrap" or something?

Reply:
Whoops, I was just overeager in my replacements.

Reply:
Actually, isn't the type attached to the fnode and not the node?